src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -100,12 +104,18 @@ const errorDetailsMap = {
   190: {
     460: new ErrorDetailsExtractorBuilder()
       .setStatus(400)
+      .setStat({
+        [tags.TAG_NAMES.ERROR_TYPE]: tags.ERROR_TYPES.ACCESS_TOKEN_EXPIRED,
+      })
       .setMessage(
         'The session has been invalidated because the user changed their password or Facebook has changed the session for security reasons',
       )
       .build(),
```

Comment:
```suggestion
    460: new ErrorDetailsExtractorBuilder()
      .setStatus(400)
      .setStat({
        [tags.TAG_NAMES.ERROR_TYPE]: tags.ERROR_TYPES.ACCESS_TOKEN_EXPIRED,
      })
      .setMessage(
        'The session has been invalidated because the user changed their password or Facebook has changed the session for security reasons',
      )
      .build(),
      
     463: new ErrorDetailsExtractorBuilder()
      .setStatus(400)
      .setStat({
        [tags.TAG_NAMES.ERROR_TYPE]: tags.ERROR_TYPES.ACCESS_TOKEN_EXPIRED,
      })
     .setMessageField('message')
      .build(),
```
Probably this is better

Reply:
WDYT @sandeepdsvs ?